### ISSUE TITLE
[FEAT] More synchronization-friendly settings

### ISSUE THREAD

**Issue by mehanoid** [st3v3nmw/obsidian-spaced-repetition#957]
**Is your feature request related to a problem? Please describe.**
In the plugin settings there is a `buryDate` property in the data.json file. The plugin writes new values for this property every day, and this may cause frequent synchronization conflicts, for example, when using git for synchronization.

**Describe the solution you'd like**
Maybe it would be better to put such volatile properties in a separate file so that you can add it to gitignore.

**Comment by 4Source** [comment]
I would agree with this. Maybe remove the ``buryDate``, ``buryList`` completely from the settings and figure it out with the stored information at the end of a card. 

For others to Test this: 
[data.json](https://github.com/st3v3nmw/obsidian-spaced-repetition/files/15415632/data.json)
[Bury.sibling.cards.md](https://github.com/st3v3nmw/obsidian-spaced-repetition/files/15415661/Bury.sibling.cards.md)

**Comment by mehanoid** [comment]
@ronzulu multiple of my own devices: laptops with obsidian-git plugin and android phone with termux and tasker.
BTW I doubt it makes sense to synchronise spaced-repetition with other people 😃

**Comment by 4Source** [comment]
> BTW I doubt it makes sense to synchronise spaced-repetition with other people 😃

I think if it is only about the settings it could be but then there can't be any flashcards related data in the data.json

**Comment by ronzulu** [comment]
> @ronzulu multiple of my own devices: laptops with obsidian-git plugin and android phone with termux and tasker. BTW I doubt it makes sense to synchronise spaced-repetition with other people 😃

Thanks for the info @mehanoid .

The reason for the question is that I remember someone saying that they were studying medicine and wanted to share notes and flashcards with their fellow students. The issue for them being that sharing files is greatly hampered because of the scheduling info embedded in the actual files.

They were after a mechanism to separate out the note/card content from their own personal scheduling info.
